### PR TITLE
cache favicon for 10 days using nextjs headers

### PR DESCRIPTION
**Pull request by Myestery**
## What is the purpose of the change:

Cache Favicon for 10 days to improve load speed

This PR uses nextjs headers to instruct the browser to cache the favicon

### Linear Task

[Linear Task URL](https://github.com/osmosis-labs/osmosis-frontend/issues/3357) a submission to close https://github.com/osmosis-labs/osmosis-frontend/issues/3357

## Brief Changelog

the nextjs config now has a `headers()` function with a list of headers

## Testing and Verifying

This change has been tested locally by rebuilding the website and verified content and links are expected

### CODE REVIEW

**Comment by Myestery** [comment]
@MaxMillington please review

**Review by jonator — APPROVED** [review]

**Comment by MaxMillington** [comment]
@Myestery We seem to be having some build issues here. Can you rebase stage and re-push? Hopefully that will fix it

**Comment by Myestery** [comment]
Okay

File: packages/web/next.config.js [hunk 1]
```diff
@@ -66,6 +66,19 @@ const config = {
...
+        source: "/favicon.ico",
+        headers: [
+          {
+            key: "Cache-Control",
+            value: "public, max-age=14400", // Cache for 10 days
+          },
+        ],
+      },
+    ];
+  },
```
[the remark this answers is not in the document]
Comment (Myestery) [reply packages/web/next.config.js]: resolved, thanks

**Comment by Myestery** [comment]
@MaxMillington I have rebased and refactored

**Review by MaxMillington — APPROVED** [review]

**Comment by Myestery** [comment]
> @Myestery I'm not sure why we can't get your build to pass. Can you try running `yarn clean` and then reinstalling and rebuilding anything? If there are any changes, push those up. PR is approved but we just want to make sure the build passes

I'll try again from my end.

**Comment by Myestery** [comment]
@MaxMillington 
I'm not really sure of whats the next step for KYC.
I tried to ask the finance team but no response yet.
Please Help

**Pull request closed without merging**